web_widget_timepicker/readme/CREDITS.rst

```diff
@@ -0,0 +1,3 @@
+* The module uses the `jquery-timepicker <https://github.com//jonthornton//jquery-timepicker#timepicker-plugin-for-jquery>`_. plugin by Jon Thornton. This software is made available under the open source MIT License. © 2014 Jon Thornton and contributors
```

Comment:
do we really need this? The std timepicker already uses bootstrap-timepicker https://github.com/odoo/odoo/blob/b6f11ace836e946eeeae110cdb08d16d80546238/addons/web/static/src/js/widgets/date_picker.js#L257